src/passes/externalContractHandler/externalContractInterfaceInserter.ts

```diff
@@ -47,8 +48,13 @@ export class ExternalContractInterfaceInserter extends ASTMapper {
   visitMemberAccess(node: MemberAccess, ast: AST): void {
     const nodeType = safeGetNodeType(node.vExpression, ast.compilerVersion);
     if (nodeType instanceof UserDefinedType && nodeType.definition instanceof ContractDefinition) {
+      // nodeType.definition will get the old ContractDefinition, see the note
+      // in storageAllocator on hotfixing getNodeType
+      // To fix this we lookup the contract definition id in the context
+      if (node.context === undefined)
+        throw new TranspileFailedError('Node context should not be undefined');
       importExternalContract(
-        nodeType.definition,
+        node.context.locate(nodeType.definition.id) as ContractDefinition,
```

Comment:
I was not aware of this hotfix/problem with getNodeType, when I made the PR #699